test/libraries/GasLimitCalculator.t.sol

```diff
@@ -5,17 +5,17 @@ import "forge-std/Test.sol";
 import {GasLimitCalculator} from "../../src/libraries/GasLimitCalculator.sol";
 
 contract GasLimitCalculatorTest is Test {
-    function test_gasLimit_100_percent() public {
+    function test_gasLimit_100_percent() public view {
```

Comment:
this file was deleted from https://github.com/Uniswap/v4-periphery/pull/268/files#

so the functions in `BibsLibrary.t.sol` will need `view` now